### PR TITLE
Add unit test for func Proto3ValuesEqual.

### PR DESCRIPTION
**Pull request by hs0210**
Signed-off-by: Hu Shuai <hus.fnst@cn.fujitsu.com>

### CODE REVIEW

File: go/sqltypes/proto3_test.go [hunk 1]
```diff
@@ -244,3 +244,80 @@ func TestQueryReponses(t *testing.T) {
...
+					Type:  1,
+					Value: []byte{0, 1, 2},
+				},
+			},
+			expected: false,
+		},
+	} {
+		if got := Proto3ValuesEqual(tc.v1, tc.v2); got != tc.expected {
+			t.Errorf("expected %v, but got %v", tc.expected, got)
+		}
```
Comment (harshit-gangal) [inline go/sqltypes/proto3_test.go:321]: change this to `require.Equal(t, tc.expected, Proto3ValuesEqual(tc.v1, tc.v2))`

**Review by harshit-gangal — CHANGES_REQUESTED** [review]

File: go/sqltypes/proto3_test.go (hunk 1 above)
Comment (hs0210) [reply go/sqltypes/proto3_test.go]: I have changed it. PTAL.

**Review by harshit-gangal — APPROVED** [review]

File: go.sum [hunk 2]
```diff
@@ -241,6 +241,7 @@ github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5y
 github.com/golang/protobuf v1.3.1/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
 github.com/golang/protobuf v1.3.2 h1:6nsPYzhq5kReh6QImI3k5qWzO4PEbvbIW2cwSfR/6xs=
 github.com/golang/protobuf v1.3.2/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
+github.com/golang/protobuf v1.4.2 h1:+Z5KGCizgyZCbGh1KZqA0fcLLkwbsjIzS4aV2v7wJX0=
 github.com/golang/snappy v0.0.1 h1:Qgr9rKW7uDUkrbSmQeiDsGa8SjGyCOGtuasMWwvp2P4=
```
Comment (harshit-gangal) [inline go.sum:245]: did you update protobuf to 1.4.2 version? PR is only related to adding test.

Comment (hs0210) [reply go.sum]: Sorry, it may my IDE automatically update it. I have recovered it.

**Pull request merged**